### ISSUE TITLE
iframe

### ISSUE THREAD

**Issue by noraj** [iknowjason/hammer#56]
Is that weird iframe supposed to be the app?

![image](https://user-images.githubusercontent.com/16578570/213869533-ae1e3718-2527-4d8b-9ff3-9f970b85626b.png)

**Comment by iknowjason** [comment]
Yes, pretty much.  It used to be a full admin dashboard.  You can just ignore the broken dashboard.  I am opening up an issue to fix the default page.  Thanks for checking in on this.
